### ISSUE TITLE
[Error] DISTROBOX_HOST_HOME incorrect when using DBX_CONTAINER_HOME_PREFIX

### ISSUE THREAD

**Issue by RyZeePy** [89luca89/distrobox#619]
**Describe the bug**
When using the `DBX_CONTAINER_HOME_PREFIX` env var or `container_home_prefix` in a config to set a home prefix for all containers, the `DISTROBOX_HOST_HOME` env var is set incorrectly inside the container. It seems to have the container name appended (i.e. /home/user/$container_name).

**To Reproduce**
```
DBX_CONTAINER_HOME_PREFIX=$HOME/dbx distrobox-create test
distrobox-enter test
echo $DISTROBOX_HOST_HOME
```
OR
```
echo 'container_home_prefix="$HOME/dbx"' > $HOME/.distroboxrc
distrobox-create test
distrobox-enter test
echo $DISTROBOX_HOST_HOME
```

**Expected behavior**
$DISTROBOX_HOST_HOME should match $HOME on the host machine.

**Desktop (please complete the following information):**
 - podman
 - podman-4.3.1
 - distrobox-1.4.2.1

**Comment by misobarisic** [comment]
Fixed in #616

**Comment by RyZeePy** [comment]
Great. Thanks @misobarisic 
Closing this one.
